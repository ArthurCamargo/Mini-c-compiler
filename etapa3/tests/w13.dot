digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 -> n4;
n2 -> n5;
n4;
n4 [label="v"];
n5;
n5 [label="3"];
n2 [label="[]"];
n3;
n3 [label="3"];
n1 [label=">>"];
n0 [label="f"];

}