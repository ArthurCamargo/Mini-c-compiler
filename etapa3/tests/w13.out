0x55a4439f53c0
0x55a4439f53c0, 0x55a4439f5540
0x55a4439f5540
0x55a4439f5540, 0x55a4439f54d0
0x55a4439f5540, 0x55a4439f5590
0x55a4439f54d0
0x55a4439f54d0, 0x55a4439f5430
0x55a4439f54d0, 0x55a4439f5480
0x55a4439f5430
0x55a4439f5430 [label="v"];
0x55a4439f5480
0x55a4439f5480 [label="3"];
0x55a4439f54d0 [label="[]"];
0x55a4439f5590
0x55a4439f5590 [label="3"];
0x55a4439f5540 [label=">>"];
0x55a4439f53c0 [label="f"];
